Notify/Notify/Notify/ViewModels/LocationSettingsPageViewModel.cs

```diff
@@ -172,6 +170,36 @@ private async void onBackButtonClicked()
 
         #endregion
 
+        #region Use_Current_Location
+
+        private bool m_IsUseCurrentLocation;
+        private string m_TmpLongitude;
+        private string m_TmpLatitude;
+        public bool IsUseCurrentLocation
+        {
+            get => m_IsUseCurrentLocation;
+            set
+            {
+                m_IsUseCurrentLocation = value;
+                OnPropertyChanged(nameof(IsUseCurrentLocation));
+
+                if (m_IsUseCurrentLocation)
+                {
+                    m_TmpLongitude = Longitude;
+                    m_TmpLatitude = Latitude;
+                    getCurrentLocationButtonClicked();
+                }
+                else
+                {
+                    Longitude = m_TmpLongitude;
+                    Latitude = m_TmpLatitude;
+                }
+            }
+        }
+        public bool m_IsUseOtherLocation => !m_IsUseCurrentLocation;
```

Comment:
There's a convertor for this kind of things.
You can ask @LinMoskovitch for how you can do it.

Reply:
When using the converter the 'IsEnabled' is not working properly - the fields only change color to gray but they are still functional and responding.